src/mps/mps.jl

```diff
@@ -26,30 +26,32 @@ mutable struct MPS
 
   MPS(N::Int) = MPS(N,fill(ITensor(),N),0,N+1)
 
-  function MPS(N::Int, 
-               A::Vector{<:ITensor}, 
-               llim::Int=0, 
+  function MPS(N::Int,
+               A::Vector{<:ITensor},
+               llim::Int=0,
                rlim::Int=N+1)
     new(N,A,llim,rlim)
   end
 end
 
-function MPS(sites)
-  N = length(sites)
-  v = Vector{ITensor}(undef, N)
-  l = [Index(1, "Link,l=$ii") for ii=1:N-1]
-  for ii in eachindex(sites)
-    s = sites[ii]
-    if ii == 1
-      v[ii] = ITensor(l[ii], s)
-    elseif ii == N
-      v[ii] = ITensor(l[ii-1], s)
-    else
-      v[ii] = ITensor(l[ii-1],s,l[ii])
-    end
-  end
-  return MPS(N,v,0,N+1)
-end
+function MPS(::Type{T},sites) where {T<:Number}
```

Comment:
The formatting of this function looks off, could you please clean it up? Thanks!

Reply:
Yes, don't know what happened there...